model/metrics/process-metrics.yaml

```diff
@@ -0,0 +1,136 @@
+    brief: "Difference in process.cpu.time since the last measurement, divided by the elapsed time and number of CPUs available to the process."
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: process.cpu.state
+
+  - id: metric.process.memory.usage
+    type: metric
+    metric_name: process.memory.usage
+    brief: "The amount of physical memory in use."
+    instrument: updowncounter
+    unit: "By"
+    attributes: []
+
+  - id: metric.process.memory.virtual
+    type: metric
+    metric_name: process.memory.virtual
+    brief: "The amount of committed virtual memory."
+    instrument: updowncounter
+    unit: "By"
+    attributes: []
+
+  - id: metric.process.disk.io
+    type: metric
+    metric_name: process.disk.io
+    prefix: process.disk
+    brief: "Disk bytes transferred."
+    instrument: counter
+    unit: "By"
+    attributes:
+      - id: process.disk.io.direction
+        brief: "The direction of the data transfer for this data point."
+        type:
+          allow_custom_values: true
+          members:
+            - id: read
+              value: 'read'
+            - id: write
+              value: 'write'
+
+  - id: metric.process.network.io
+    type: metric
+    metric_name: process.network.io
+    brief: "Network bytes transferred."
+    instrument: counter
+    unit: "By"
+    attributes:
+      - id: process.network.io.direction
+        brief: "The direction of the data transfer for this data point."
+        type:
+          allow_custom_values: true
+          members:
+            - id: receive
+              value: 'receive'
+            - id: transmit
+              value: 'transmit'
+
+  - id: metric.process.threads
+    type: metric
+    metric_name: process.threads
+    brief: "Process threads count."
+    instrument: updowncounter
+    unit: "{thread}"
+    attributes: []
+
+  - id: metric.process.open_file_descriptors
+    type: metric
+    metric_name: process.open_file_descriptors
+    brief: "Number of file descriptors in use by the process."
+    instrument: updowncounter
+    unit: "{count}"
+    attributes: []
+
+  - id: metric.process.context_switches
+    type: metric
+    metric_name: process.context_switches
+    brief: "Number of times the process has been context switched."
+    instrument: counter
+    unit: "{count}"
+    attributes:
+      - id: process.context_switches.type
+        brief: "Specifies whether the context switches for this data point were voluntary or involuntary."
+        type:
+          allow_custom_values: true
+          members:
+            - id: voluntary
+              value: 'voluntary'
+            - id: involuntary
+              value: 'involuntary'
+
+  - id: metric.process.paging.faults
+    type: metric
+    metric_name: process.paging.faults
+    brief: "Number of page faults the process has made."
+    instrument: counter
+    unit: "{fault}"
+    attributes:
+      - id: process.paging.faults.type
```

Comment:
I opened an issue to discuss it #394. I noticed the decision is also not documented anywhere and we also have the other discussion about "metric names should be allowed to be namespaces" https://github.com/open-telemetry/semantic-conventions/issues/50

Reply:
Maybe going with sth like `process.paging.fault.count` and `process.paging.fault.type` would be more tidy?
This would also be aligned with ECS' alignment effort and not having fields conflicting with leaf nodes as described at https://github.com/open-telemetry/semantic-conventions/pull/333#issuecomment-1764384113.